src/main/java/appeng/menu/me/crafting/CraftingStatus.java

```diff
@@ -26,70 +26,32 @@
 import net.minecraft.network.FriendlyByteBuf;
 
 import appeng.crafting.execution.CraftingCpuLogic;
-import appeng.crafting.execution.ElapsedTimeTracker;
 import appeng.menu.me.common.IncrementalUpdateHelper;
 
 /**
  * Describes a currently running crafting job. A crafting status can either be a full update which replaces any
- * previously kept state on the client ({@link #isFullStatus()}, or an incremental update, which uses previously sent
+ * previously kept state on the client ({@link #fullStatus ()}, or an incremental update, which uses previously sent
  * {@link CraftingStatusEntry#getSerial() serials} to update entries on the client that were previously sent. To reduce
  * the packet size for updates, the {@link CraftingStatusEntry#getWhat() stack} for entries that were previously sent to
  * the client are set to {@code null}.
+ *
+ * @param fullStatus         True if this status update replaces any previous status information. Otherwise it should be
+ *                           considered an incremental update.
+ * @param elapsedTime
+ * @param remainingItemCount
+ * @param startItemCount
  */
-public class CraftingStatus {
-
-    public static final CraftingStatus EMPTY = new CraftingStatus(true, 0, 0, 0, Collections.emptyList());
+public record CraftingStatus(boolean fullStatus, long elapsedTime, long remainingItemCount, long startItemCount,
```

Comment:
As discussed in call, please revert the record change to maintain compat and  then I'll review the pausing part of this PR :)